server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -3002,6 +3009,24 @@ public TServerInstance getTabletSession() {
     }
   }
 
+  private void checkWalCanSync(ServerContext context) {
+    VolumeChooserEnvironment chooserEnv =
+        new VolumeChooserEnvironmentImpl(VolumeChooserEnvironment.ChooserScope.LOGGER, context);
+    String logPath = fs.choose(chooserEnv, ServerConstants.getBaseUris(context)) + Path.SEPARATOR
+        + ServerConstants.WAL_DIR;
+    if (!fs.canSyncAndFlush(new Path(logPath))) {
+      // sleep a few seconds in case this is at cluster start...give monitor
+      // time to start so the warning will be more visible
+      try {
```

Comment:
Could use `UtilWaitThread.sleep()` its an internal Accumulo util

Reply:
Ooh, thanks.  I'll fix this.